core/src/display_object.rs

```diff
@@ -1166,17 +1179,29 @@ pub trait TDisplayObject<'gc>:
     /// This differs from the bounds that are exposed to Flash, in two main ways:
     /// - It may be larger if filters are applied which will increase the size of what's shown
     /// - It does not respect scroll rects
-    fn render_bounds_with_transform(&self, matrix: &Matrix) -> Rectangle<Twips> {
+    fn render_bounds_with_transform(
+        &self,
+        matrix: &Matrix,
+        include_own_filters: bool,
+        view_matrix: &Matrix,
+    ) -> Rectangle<Twips> {
```

Comment:
Function signature changes without doc comment changes are always _sus_, but in this case, I think the same comment still applies and is sufficient.

Reply:
I wrote that comment just the other day whilst thinking about this change 😛